tests/components/test_rflink.py

```diff
@@ -10,23 +10,19 @@
     ATTR_ENTITY_ID, SERVICE_TURN_OFF, SERVICE_STOP_COVER)
 
 
-@asyncio.coroutine
-def mock_rflink(hass, config, domain, monkeypatch, failures=None):
-    """Create mock Rflink asyncio protocol, test component setup."""
+async def mock_rflink(hass, config, domain, monkeypatch, failures=None):
+    """Create mock RFLink asyncio protocol, test component setup."""
     transport, protocol = (Mock(), Mock())
 
-    @asyncio.coroutine
-    def send_command_ack(*command):
+    async def send_command_ack(*command):
         return True
     protocol.send_command_ack = Mock(wraps=send_command_ack)
 
-    @asyncio.coroutine
-    def send_command(*command):
+    async def send_command(*command):
```

Comment:
This shouldn't be a coroutine. It's a callback in the library.

Reply:
I see your point.
Changing the sentence to `def send_command(*command)` without `async` or notation, passes the tests correctly.
Thanks.